packages/lib/src/components/PayByBankUS/PayByBankUS.tsx

```diff
@@ -0,0 +1,75 @@
+import { h } from 'preact';
+import { CoreProvider } from '../../core/Context/CoreProvider';
+import RedirectElement from '../Redirect';
+import RedirectButton from '../internal/RedirectButton';
+import { TxVariants } from '../tx-variants';
+import './PayByBankUS.scss';
+import getIssuerImageUrl from '../../utils/get-issuer-image';
+
+export default class PayByBankUS extends RedirectElement {
+    public static type = TxVariants.paybybank_AIS_DD;
+
+    /* TODO
+    DONE render component over PbB US
+    DONE - show correct disclaimer messages
+    TODO - upload translations  
+    TODO - fix logo
+    DONE - display brands
+    DONE - deduplicate css classes
+    TODO - check if needs clean/refactor
+    */
```

Comment:
Is it missing just the translations for now or is there more stuff?

Reply:
Translations and logos, however it's getting stored payment added too.